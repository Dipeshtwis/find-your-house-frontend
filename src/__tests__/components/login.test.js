/* eslint-disable no-unused-vars */
import React from 'react';
import renderer from 'react-test-renderer';
import login from '../../components/Login';

it('renders login component correctly', () => {
  const login = renderer.create(<login />).toJSON();
  expect(login).toMatchSnapshot();
});
