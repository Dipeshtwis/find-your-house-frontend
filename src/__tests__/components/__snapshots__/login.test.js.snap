// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders login component correctly 1`] = `<login />`;
